scripts/sort-makefile-am-sources.pl

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/perl
+#
+## Copyright (C) 1996-2020 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+use strict;
+use warnings;
+
+while (<>) {
+    if (m!^#!) {
+        print;
+        next;
+    }
+    if (/^(\S+_SOURCES)\s*=\s*\\$/) {
+        print "$1 = \\\n";
+    } else {
+        print;
+        next;
+    }
+    # accumulate files and prep for sorting
+    my %files;
```

Comment:
I am not sure this would happen, but I would avoid (the impression of) using uninitialized variables:

```suggestion
    my %files = ();
```

Reply:
sure